src/dune_rules/melange_stanzas.ml

```diff
@@ -47,6 +48,12 @@ module Emit = struct
     t
 
   let decode =
+    let extension_field name =
+      let+ loc, extension = located (field name ~default:"js" string) in
```

Comment:
The location isn't quite correct. It should span the field's value rather than the entire field. So:

```ocaml
field name ~default:"js" (located string)
```

Reply:
https://github.com/ocaml/dune/pull/6457/commits/d773009509236c67b849c00b5140aacdab34cc73